tests/ruby/RuboCopBearTest.py

```diff
@@ -40,3 +41,33 @@ def bad_name
                                 valid_files=(camelCaseFile,),
                                 invalid_files=(snake_case_file,),
                                 settings={"rubocop_config": rubocop_config})
+
+# Testing settings
+good_file = """
+def goodindent
+ # 1 space indent
+end
+""".splitlines(keepends=True)
+
+bad_file = """
+def badindent
+  # 2 spaces indent
+end
+""".splitlines(keepends=False)
+
+RuboCopBearSettingsTest = verify_local_bear(
+                              RuboCopBear,
+                              valid_files=(good_file,),
+                              invalid_files=(bad_file,),
+                              settings={"indentation_width": 1})
+bad_file = """
+'using single-quotes'
+""".splitlines(keepends=True)
+good_file = """
+"using double-quotes"
+""".splitlines(keepends=True)
+RuboCopBearSettingsTest = verify_local_bear(
+                              RuboCopBear,
+                              valid_files=(good_file,),
+                              invalid_files=(bad_file,),
+                              settings={"string_literals": "double_quotes"})
```

Comment:
what do you suggest? i can drop this tests actually since i tested for settings earlier, but i didn't feel good by testing just one setting, or i can mix thee two settings in the earlier example?